### PR TITLE
Cleaning up some of the hardcoded lists to much

### PR DESCRIPTION
**Pull request by Loaded4Bears**
cleaner concrete_typesof

Attempting to clean up some hardcoded lists that can be easily simplified, expanded the cola and booze spawner, fixed icon for the pizza and cola spawners.

## Why's this needed? 

Toooo many hardcoded lists

### CODE REVIEW

File: code/obj/random_spawners.dm [hunk 1]
```diff
@@ -1925,9 +1866,7 @@
 	//only 1 can spawn for now since the pool size is small. Might want to increase it if the pool size increases by a fair amount
 
 	items2spawn = list(/obj/item/aiModule/experimental/equality/a,
-						/obj/item/aiModule/experimental/equality/b,
-						/obj/item/aiModule/experimental/corrupted,
-						/obj/item/aiModule/experimental/historic)
```
Comment (JORJ949) [inline code/obj/random_spawners.dm:1930]: Outdated.

**Review by JORJ949 — CHANGES_REQUESTED** [review]
If I understand concrete_typesof properly then this removes and adds stuff to these spawners, such as syndicate tables, old medkits and various other things. Probably best leaving these as lists so you dont end up with /obj/item/clothing/mask/gas/swat/syndicate in an otherwise clean random gimmick mask spawner, or any /obj/item/reagent_containers/food/drinks/bottle/poison_that_instagibs_everything (example) in a random booze spawner.

**Review by Loaded4Bears — COMMENTED** [review]
Reverted the first aid kits, masks, shoes, guardbot tools. 

Mines should be ok, pizza looks ok, the 'bad' booze is under /food/drinks and the 'regular shelf' stuff is under food/drinks/bottle.

Furniture parts and gimmick clothing doesn't seem to have anything traitor/secret/OP

**Review by Glamurio — CHANGES_REQUESTED** [review]
Always happy for refactoring, however this isn't a simple refactor, you're changing a lot of core functionality of some spawners. For some of them that's fine, but for some others that would really change their function.

Also, `SpaceStop.dmm` is a new prefab, please atomize this PR and make a separate one for the map or the cleanup, but not both at once.

**Comment by JORJ949** [comment]
> 
> Furniture parts and gimmick clothing doesn't seem to have anything traitor/secret/OP

You should also consider future items that could be added there, I think the set list or a var as to "do i spawn in random spawner" (like how surplus crates work) would be better than just anything of this type.

**Review by Loaded4Bears — COMMENTED** [review]
didn't mean to include the new map, i'll close this entire pull and come back for a  2nd try.

**Pull request closed without merging**